nixos/doc/manual/release-notes/rl-2211.section.md

```diff
@@ -1,27 +1,53 @@
 # Release 22.11 (“Raccoon”, 2022.11/??) {#sec-release-22.11}
 
-Support is planned until the end of June 2023, handing over to 23.05.
+The NixOS release team is happy to announce a new version of NixOS 22.11. NixOS is both a linux distribution, and a set of packages usable on other Linux systems and macOS.
```

Comment:
I'm not sure if we want this, if only because we have similar stuff in the blog post.

Reply:
I'm not opposed.